gef-ui/templates/includes/application-preview/task-comments.njk

```diff
@@ -5,12 +5,12 @@
     {% set bannerHtml %}
 
     {% if renderReviewDecisionLink %}
-      <div class="govuk-!-margin-bottom-6" data-cy="ukef-review">
-        <a href="/gef/application-details/{{ dealId }}/review-decision" class="govuk-link">Review UKEF decision</a>
+      <div class="govuk-!-margin-bottom-0" data-cy="ukef-review">
+        <a href="/gef/application-details/{{ dealId }}/review-decision" class="govuk-link" data-cy="ukef-review-link">Review UKEF decision</a>
       </div>
     {% endif %}
 
-    {% if unissuedFacilitiesPresent and facilitiesChangedToIssued.length === 0 %}
+    {% if unissuedFacilitiesPresent and facilitiesChangedToIssued.length === 0 and ukefDecisionAccepted %}
```

Comment:
Should we move this condition into  the server side so the template is simpler? Got 3 conditionals in the template now

Reply:
Have moved to `gef-ui` controller.

Reply:
Have moved to `gef-ui` application-details.